src/objecttypes/conf/base.py

```diff
@@ -53,30 +52,6 @@
 # Django setup configuration
 #
 SETUP_CONFIGURATION_STEPS = [
-    "objecttypes.config.site.SiteConfigurationStep",
-    "objecttypes.config.objects.ObjectsAuthStep",
-    "objecttypes.config.demo.DemoUserStep",
+    "objecttypes.setup_configuration.steps.SitesConfigurationStep",
```

Comment:
This is still undergoing some discussion, but we might move this step to `django-setup-configuration.contrib` to avoid having to create it in every project